### DISCUSSION TITLE
Filter snippets by tag -> pet list -t $tag

### DISCUSSION THREAD

**Discussion by alxgu** [knqyf263/pet#312] [Ideas]
It would be great, when the `list` command would acccept `-t $tag_name` to show only snippets which are tagged with `$tag_name`

**Comment by RamiAwar** [comment]
Great suggestion! Would definitely accept a PR that does this 🙂

If anyone is interested in doing this, I'll gladly support.

**Comment by ashupatial** [comment]
It would be a useful enhancement to allow the list command to accept a -t $tag_name flag, enabling users to filter snippets based on specific tags. This would improve usability and organization, especially when managing a large number of snippets. If this feature isn't currently available, consider opening a feature request with the maintainers of the tool.

**Comment by alxgu** [comment]
Thank you @a2xdeveloper for implementing the feature request :slightly_smiling_face: 

https://github.com/knqyf263/pet/pull/334
